public/app/plugins/datasource/stackdriver/components/SimpleSelect.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { SFC } from 'react';
+
+interface Props {
+  onValueChange: (e) => void;
+  options: any[];
+  value: string;
+  label: string;
+}
+
+const SimpleSelect: SFC<Props> = props => {
```

Comment:
Think we already have a similar component, https://github.com/grafana/grafana/blob/master/public/app/core/components/Picker/SimplePicker.tsx

Reply:
The only problem that I can see with using this component is that only a part of the whole page, in this case the stackdriver query editor, would get the look and feel of the new select option, whereas the rest of the page has the look and feel of the old angularjs select option. Wouldn't it be better to use the old style of the select until the whole template page is migrated to react?